### PR TITLE
task2 login page

### PR DESCRIPTION
**Pull request by poberezhniuk**

### CODE REVIEW

File: src/app/in-memory-data.service.ts [hunk 1]
```diff
@@ -33,4 +33,8 @@ export class InMemoryDataService implements InMemoryDbService {
 
     return { users };
   }
+
+  // genId(users: User[]): number {
```
Comment (alexandr2134) [inline src/app/in-memory-data.service.ts]: remove commented code or describe some TODO

File: src/app/login/login.component.ts [hunk 2]
```diff
@@ -1,15 +1,62 @@
...
+  get loginControls() {
+    return this.loginForm.controls;
+  }
+
+  onLogin() {
+    // If userlist empty abdon operation
+    if (!this.userList.length) return;
+    // search user with same email and pass
+    if (this.checkEmailAndPassword()) {
+      console.log(true);
```
Comment (alexandr2134) [inline src/app/login/login.component.ts]: remove logs

File: src/app/reducers/index.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+export interface State {
+
+}
+
+export const reducers: ActionReducerMap<State> = {
+
+};
+
+
+export const metaReducers: MetaReducer<State>[] = !environment.production ? [] : [];
```
Comment (alexandr2134) [inline src/app/reducers/index.ts]: always try to go from "true" value. in this case check "environment.production ? value1 : value2"

File: src/app/user-list/user-list.component.ts [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
+  styleUrls: ["./user-list.component.scss"]
 })
 export class UserListComponent implements OnInit {
-
-  constructor() { }
+  userList;
+  constructor(private userService: UserService) {}
 
   ngOnInit() {
+    this.userService.getUsers().subscribe(users => (this.userList = users));
```
Comment (alexandr2134) [inline src/app/user-list/user-list.component.ts:14]: you don't need parentheses here "(this.userList = users)"

File: src/app/login/login.component.ts (hunk 2 above)
Comment (poberezhniuk) [reply src/app/login/login.component.ts]: it's test vertion
replace them later, when implement "login" status in ngrx

File: src/app/reducers/index.ts (hunk 3 above)
Comment (poberezhniuk) [reply src/app/reducers/index.ts]: replace them later, when implement "login" status in ngrx

File: src/app/user-list/user-list.component.ts (hunk 4 above)
Comment (poberezhniuk) [reply src/app/user-list/user-list.component.ts:14]: its not me, auto-prettier make it)

File: src/app/in-memory-data.service.ts (hunk 1 above)
Comment (poberezhniuk) [reply src/app/in-memory-data.service.ts]: removed

File: src/app/login/login.component.ts (hunk 2 above)
Comment (poberezhniuk) [reply src/app/login/login.component.ts]: done

**Pull request merged**